test/kixi/integration/base.clj

```diff
@@ -199,15 +203,23 @@
     pfr))
 
 (defn post-file
+  ([uid file-name]
+   (post-file uid uid file-name nil))
   ([uid file-name schema-id]
    (post-file uid uid file-name schema-id))
   ([uid ugroup file-name schema-id]
-   (post-file-and-wait :file-name file-name 
-                       :schema-id schema-id 
-                       :user-id uid
-                       :user-groups ugroup
-                       :sharing {:file-read [ugroup]
-                                 :meta-read [ugroup]})))
+   (if schema-id
+     (post-file-and-wait :file-name file-name 
```

Comment:
there is already a test in post-file-and-wait, so it'd be ok to pass a nul in schema-id here?